run-lint.sh

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env bash
-
-set -euo pipefail
-
-[[ -n ${DEBUG:-} ]] && set -x
-
-gopath="$(go env GOPATH)"
-
-if ! [[ -x "$gopath/bin/golangci-lint" ]]; then
-	echo >&2 'Installing golangci-lint'
-	curl --silent --fail --location \
-		https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$gopath/bin" v1.43.0
-fi
-
-# configured by .golangci.yml
-"$gopath/bin/golangci-lint" run
```

Comment:
how are we going to run locally?

Reply:
Does Go have dev dependencies in `go.mod`? I'm searching for it in Google and I don't think so. 🤔 

By the way, when I've set up `"go.lintTool": "golangci-lint",` in VS Code, it automatically installed the tool.

Want me to add this file back? This was primarily used in the CI and I don't know if it makes sense to have it anymore.

Reply:
without this, hard to test locally

make is better of course, you can copy some parts from server

Reply:
added a `Makefile`